Dockerfile

```diff
@@ -33,7 +33,8 @@ RUN set -x \
  && /fossology/utils/fo-installdeps -e -y \
  && $_cleanup
 RUN curl -sS https://getcomposer.org/installer | php && \
-    mv composer.phar /usr/local/bin/composer
+	mkdir /tmp/composer && \
+    mv composer.phar /tmp/composer/composer
```

Comment:
the path `/tmp/composer/composer` is not a very good choice, since 
- this folder gets cleaned on every reboot on most systems
- it might interfere with other fossoloy development setups
- `/tmp/` might be mounted with `noexec` for security reasons

Note: this comment is to late and is related to https://github.com/fossology/fossology/pull/701 which is already merged. In my opinion one should change [`COMPOSERTMP = /tmp/composer`](https://github.com/fossology/fossology/pull/701/files#diff-f57f2991a6aa25fe45d8036c51bf8b4dR7) to something in the current subtree.